src/libs/ReportUtils.js

```diff
@@ -701,6 +700,15 @@ function getIOUReportActionMessage(type, total, participants, comment, currency)
         case CONST.IOU.REPORT_ACTION_TYPE.SPLIT:
             iouMessage = `Split ${amount} with ${from}${comment && ` for ${comment}`}`;
             break;
+        case CONST.IOU.REPORT_ACTION_TYPE.CANCEL:
+            iouMessage = `Cancelled the ${amount} request${comment && ` for ${comment}`}`;
+            break;
+        case CONST.IOU.REPORT_ACTION_TYPE.DECLINE:
+            iouMessage = `Declined the ${amount} request${comment && ` for ${comment}`}`;
+            break;
+        case CONST.IOU.REPORT_ACTION_TYPE.PAY:
+            iouMessage = `Paid ${amount} to ${from}${comment && ` for ${comment}`}`;
```

Comment:
With this message, can you ever pay more people at once (now). It is always just 1:1 right?

Reply:
I think we can only pay one report at a time and even when we split a bill we create multiple 1:1 iouReports, so I think this message is ok?

Reply:
Ah the `from` is just tripping me up in this function, I maybe some comments showing what the values of `from` would be could be helpful to understand this method better.

Reply:
renamed the variable to `who` which is what we use in PHP